trino/dbapi.py

```diff
@@ -72,16 +84,6 @@
 logger = trino.logging.get_logger(__name__)
 
 
-def connect(*args, **kwargs):
-    """Constructor for creating a connection to the database.
-
-    See class :py:class:`Connection` for arguments.
-
-    :returns: a :py:class:`Connection` object.
-    """
-    return Connection(*args, **kwargs)
-
```

Comment:
extract no-op movements to a single commit

Reply:
Reverted. Used [Postponed Evaluation of Annotations](https://peps.python.org/pep-0563/) instead of reordering class definition. Reason was that type hint contained names that have not been defined yet.